Python/Product/PythonTools/PythonTools/Options/PythonDebuggingOptionsControl.cs

```diff
@@ -13,13 +13,21 @@
 //
 // See the Apache Version 2.0 License for specific language governing
 // permissions and limitations under the License.
-
+using Microsoft.PythonTools.Debugger;
+using System;
+using System.Collections.Generic;
 using System.Windows.Forms;
 
 namespace Microsoft.PythonTools.Options {
     public partial class PythonDebuggingOptionsControl : UserControl {
         public PythonDebuggingOptionsControl() {
             InitializeComponent();
+
+            // set allowed values for the variable presentation options
+            var varPresComboBoxes = new List<ComboBox> { _varPresClassComboBox, _varPresFunctionComboBox, _varPresProtectedComboBox, _varPresSpecialComboBox };
+            foreach (var varPresComboBox in varPresComboBoxes) {
+                varPresComboBox.DataSource = Enum.GetValues(typeof(PresentationMode));
```

Comment:
should add localization.

Reply:
The labels are already in the resx, so they will be localized. But are you saying I should localize the values in the enum? Those values need to be passed to the debugger verbatim.

Reply:
sorry. i thought the enums were being read into the labels.